### ISSUE TITLE
Console templates not working in Firefox

### ISSUE THREAD

**Issue by StephanErb** [prometheus/prometheus#1442]
It seems like console templates in Prometheus 16.2 are not working in Firefox 44 due to a date parsing issue:

A console template (when clicking on the rewind butten) runs the following code:
```
PromConsole.TimeControl.prototype.getEndDate = function() {
  if (this.endElement.value === '') {
    return null;
  }
  return new Date(this.endElement.value).getTime();
};
```
The property `this.endElement.value` is set to a value such as `2016-3-1 17:55`. It seems like Firefox is unable to create a valid Date from this and returns `None` from the `getEndDate` function. This breaks the template rendering.

Steps to reproduce:
* Open http://demo.robustperception.io:9090/consoles/node-overview.html?instance=demo.robustperception.io%3a9100 in Firefox
* Click the rewind button
* The URL will append the invalid paramters `#pctc{%22duration%22%3A3600%2C%22endTime%22%3Anull}` to the URL. 
* Not timeseries are shown.

**Comment by juliusv** [comment]
@StephanErb Thanks!

@brian-brazil Besides not zero-padding month and day, `PromConsole.TimeControl.prototype.getHumanDate` does not insert a `T` between date and time that's necessary for the resulting string to be parsed as a valid `Date` in Firefox. If I replace the entire function simply by `return date.toISOString();`, it works. The question is whether that is "human-readable" enough for you, as the output in the end-time input box now looks like this: `2016-03-02T09:32:02.835Z`. Probably ok? Or do we need another layer that translates between what's actually shown in the input box vs. what gets used internally for the `Date` conversion?

**Comment by brian-brazil** [comment]
The expression browser also produces times like that, so what we probably want to do is use unixtime in the URL fragment.

**Comment by fabxc** [comment]
Can we exclude console templates from 1.0?
It doesn't seem anyone is working on them and they'll just block us. Arguably they are not part of Prometheus server directly. That they happen to be placed in the Prometheus repository is unideal.

**Comment by brian-brazil** [comment]
The example templates are excluded from the 1.0 guarantee.

**Comment by fabxc** [comment]
Cool, can we remove the 1.0 milestone then? As in the other issue, I'd like
to remove anything that defers 1.0.

**Comment by brian-brazil** [comment]
It's something we should really fix before 1.0.
